src/vs/workbench/contrib/chat/browser/actions/chatContextActions.ts

```diff
@@ -216,15 +227,28 @@ class AttachContextAction extends Action2 {
 					isDynamic: true
 				});
 			} else if (pick && typeof pick === 'object' && 'resource' in pick && pick.resource) {
-				// #file variable
-				toAttach.push({
-					...pick,
-					id: this._getFileContextId({ resource: pick.resource }),
-					value: pick.resource,
-					name: pick.label,
-					isFile: true,
-					isDynamic: true
-				});
+				if (/\.(png|jpg|jpeg)$/i.test(pick.resource.path)) {
```

Comment:
Little confused by what images we support where, there is png above, png+jpg here, png+jpg+bmp+gif+tiff below?